src/Compilers/CSharp/Portable/Binder/ForEachLoopBinder.cs

```diff
@@ -1110,11 +1110,7 @@ private EnumeratorResult SatisfiesIEnumerableInterfaces(ExpressionSyntax collect
                         extensionReceiverOpt: null,
                         expanded: false,
                         collectionExpr.Syntax,
-                        diagnostics,
-                        // C# 8 shipped allowing the CancellationToken of `IAsyncEnumerable.GetAsyncEnumerator` to be non-optional,
-                        // filling in a default value in that case. https://github.com/dotnet/roslyn/issues/50182 tracks making
-                        // this an error and breaking the scenario.
-                        assertMissingParametersAreOptional: false);
```

Comment:
>assertMissingParametersAreOptional [](http://example.com/codeflow?start=24&length=34)

@jjonescz Can we get rid of this parameter or there are still consumers that pass `false`?